get-random-throw.js

```diff
@@ -1,3 +1,13 @@
-export function getRandomThrow () {
-    
-}
\ No newline at end of file
+export function getRandomThrow() {
+    let value = Math.ceil(Math.random() * 3);
+
+    if (value === 1) {
+        return value = 'rock';
+
+    } else if (value === 2) {
+        return value = 'paper';
+
+    } else {
```

Comment:
probably could have done this without the 'else', since it needs to hit one of the three cases